scripts/benchmark_bolt.py

```
@@ -0,0 +1,99 @@
+        if i % 4 == 0:
+            skills += " TTR"
+
+        c.execute(
+            "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+            (f"CREW_{i}", role, skills, "42h"),
+        )
+
+        crew_id = c.lastrowid
+        # Half are available
+        if i % 2 == 0:
+            now = datetime.now()
```

Ruff (0.15.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)